### PR TITLE
Linting and travis

### PR DESCRIPTION
**Pull request by sanjeevghimire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/IBM/<repo>.svg?branch=master)](https://travis-ci.org/IBM/<repo>)
+[![Build Status](https://travis-ci.org/IBM/.svg?branch=master)](https://travis-ci.org/IBM/<repo>)
```
Comment (stevemar) [inline README.md]: this should be:

[Build Status](https://travis-ci.org/IBM/snap-and-translate)

File: README.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Install the [pre-requisites](https://github.com/IBM/container-service-getting-st
 * Set the Kubernetes environment to work with your cluster:
 
 ```
-$ bx cs cluster-config <replace_with_your_cluster_name>
+$ ibmcloud cs cluster-config <replace_with_your_cluster_name>
```
Comment (stevemar) [inline README.md:80]: i don't actually know if this was a clean replace for `ibmcloud` to `bx`, i'd recommend someone try these commands out.

**Review by stevemar — COMMENTED** [review]
thanks for adding those changes @RiyaMRoy04

File: README.md (hunk 2 above)
Comment (RiyaMRoy04) [reply README.md:80]: I did try the command :) @stevemart

Comment (sanjeevghimire) [reply README.md:80]: both `bx` and `ibmcloud` works.

Comment (RiyaMRoy04) [reply README.md:80]: screen shot 2018-09-17 at 9 28 21 pm preview of it :)

Comment (stevemar) [reply README.md:80]: nice! :)

**Review by stevemar — APPROVED** [review]
thanks @sanjeevghimire and @RiyaMRoy04 !

**Pull request merged**